app/lib/common/widgets/blinking_text.dart

```diff
@@ -0,0 +1,204 @@
+import 'package:flutter/material.dart';
+
+class BlinkText extends StatefulWidget {
```

Comment:
`BlinkText`
This entire class is directly copied from https://pub.dev/packages/blinking_text

Reply:
If we are not including it as a proper package, we still must adhere to its license then. see above.